### PR TITLE
Changed command for building node modules to npm ci

### PR DESCRIPTION
**Pull request by ddc22**
To maintain the exact tree as committed in development npm has introduced the npm ci command which is meant to provide reproducible builds.
please take a look at the [official documentation](https://docs.npmjs.com/cli/ci.html) for more information.

### CODE REVIEW

**Comment by fhemberger** [comment]
Hi, thanks for your contribution!

There are a couple more instances of `npm install` which should be `npm ci` according to the definition. It would also be nice if you could add a sentence explaining the difference to `npm install`, as not everybody will be familiar with this command. A link to the npm docs would help as well.

**Comment by ddc22** [comment]
Hi @fhemberger! Glad to contribute :). Please let me know if there should be any other additions.

File: locale/en/docs/guides/nodejs-docker-webapp.md [hunk 1]
```diff
@@ -110,13 +110,15 @@ COPY package*.json ./
...
-# RUN npm install --only=production
+# RUN npm ci --only=production
 ```
 
 Note that, rather than copying the entire working directory, we are only copying
 the `package.json` file. This allows us to take advantage of cached Docker
 layers. bitJudo has a good explanation of this
-[here](http://bitjudo.com/blog/2014/03/13/building-efficient-dockerfiles-node-dot-js/).
+[here](http://bitjudo.com/blog/2014/03/13/building-efficient-dockerfiles-node-dot-js/). 
+Furthermore, the npm ci command, specified in the comments, helps provide faster, reliable, reproducible builds for production environments.
```
Comment (ZYSzys) [inline locale/en/docs/guides/nodejs-docker-webapp.md]: Nit:
```suggestion
Furthermore, the `npm ci` command, specified in the comments, helps provide faster, reliable, reproducible builds for production environments. 
```

**Review by ZYSzys — APPROVED** [review]
LGTM

File: locale/en/docs/guides/nodejs-docker-webapp.md [hunk 2]
```diff
@@ -110,13 +110,15 @@ COPY package*.json ./
 
 RUN npm install
```
Comment (fhemberger) [inline locale/en/docs/guides/nodejs-docker-webapp.md:111]: Should also be changed here.

File: locale/en/docs/guides/nodejs-docker-webapp.md [hunk 3]
```diff
@@ -156,7 +158,7 @@ COPY package*.json ./
...
```
Comment (fhemberger) [inline locale/en/docs/guides/nodejs-docker-webapp.md:159]: Should also be changed here.

**Review by fhemberger — CHANGES_REQUESTED** [review]

File: locale/en/docs/guides/nodejs-docker-webapp.md (hunk 2 above)
Comment (ddc22) [reply locale/en/docs/guides/nodejs-docker-webapp.md:111]: NPM CI is only required for production builds not the normal build

File: locale/en/docs/guides/nodejs-docker-webapp.md (hunk 3 above)
Comment (ddc22) [reply locale/en/docs/guides/nodejs-docker-webapp.md:159]: NPM CI is only required for production builds not the normal build

File: locale/en/docs/guides/nodejs-docker-webapp.md (hunk 1 above)
Comment (ddc22) [reply locale/en/docs/guides/nodejs-docker-webapp.md]: Added code formatting as suggested

**Review by willin — APPROVED** [review]

**Review by fhemberger — APPROVED** [review]

**Comment by fhemberger** [comment]
Thank you!

**Comment by ghost** [comment]
@nodejs/localization：Need to be translated for your own languages if possible and free for you.

**Pull request merged**